### ISSUE TITLE
Add checkbox "Unparsable values" -> Error occurred (opposite of "Parsed values")

### ISSUE THREAD

**Issue by ypid** [opening-hours/opening_hours-statistics#18]
Continuing the discussion from #17 here in a new issue. I am not sure if I want an additional check box.

@HubMiner There is currently:

* "Parsed values" -> No error occurred.
* "Warning values" -> Warnings occurred.

You now want ("Unparsable values in %" -> Error occurred)?

**Comment by HubMiner** [comment]
Right, I am looking for count of error (unparsable) values: this gives indication what is the OH backlog we need to fix.  

By the way, here is same graph from Osmose (different parser: different values).
Spikes are due to changes in the tool, not OSM data.
http://osmose.openstreetmap.fr/en/errors/graph.png?item=3250

**Comment by ypid** [comment]
All right. I am accepting your feature request ;-)
